testsuite/integration-arquillian/tests/base/pom.xml

```diff
@@ -44,6 +44,10 @@
         <mvel.version>2.4.0.Final</mvel.version>
         <systemrules.version>1.19.0</systemrules.version>
         <common.resources>${basedir}/../../servers/auth-server/jboss/common</common.resources>
+        <maven-compiler-plugin.version>3.8.1</maven-compiler-plugin.version>
+        <maven.compiler.release>11</maven.compiler.release>
+        <maven.compiler.target>11</maven.compiler.target>
+        <maven.compiler.source>11</maven.compiler.source>
```

Comment:
Is it an option to udpate them to use JDK 11? It will be the default JDK for quarkus based keycloak and the other modules have afaik the targets set accordingly (we also use JDK 11 to build upstream, see https://github.com/keycloak/keycloak/blob/main/.github/workflows/ci.yml#L14 ) - cc @nehachopra27 @vmuzikar fyi